lib/rts.c

```diff
@@ -725,11 +725,14 @@ int process_arguments(int argc, char *argv[])
       break;
 
     case 'c':
-      if (&sail_rts_set_coverage_file) {
+      #pragma clang diagnostic push
+      #pragma clang diagnostic ignored "-Wtautological-pointer-compare"
+      if (&sail_rts_set_coverage_file != NULL) {
```

Comment:
```suggestion
      if (sail_rts_set_coverage_file != NULL) {
```
I think the spurious `&` is the reason for the warning. This will call NULL if invoked with `-c` but sail was compiled without coverage.

Reply:
Ah when I changed the extern into a pointer variable I needed to remove this.

Interestingly it didn't break the coverage on tests either.